qa/suites/upgrade/telemetry-upgrade/reef-x/1-tasks.yaml

```diff
@@ -1,34 +1,51 @@
+overrides:
+  ceph:
+    log-ignorelist:
+      - mons down
+      - mon down
+      - MON_DOWN
+      - out of quorum
+      - PG_
+      - OSD_DOWN
+      - POOL_APP_NOT_ENABLED
+      - pgs degraded
+      - pg degraded
+      - object degraded
+      - objects degraded
+      - FS_DEGRADED
+      - MDS_ALL_DOWN
+      - OSD_UPGRADE_FINISHED
+      - do not have an application enabled
+      - is down
 tasks:
 - install:
-    branch: pacific
+    branch: reef
     exclude_packages:
       - ceph-volume
 - print: "**** done install task..."
-- print: "**** done start installing pacific cephadm ..."
+- print: "**** done start installing reef cephadm ..."
 - cephadm:
-    image: quay.io/ceph/daemon-base:latest-pacific
-    cephadm_branch: pacific
-    cephadm_git_url: https://github.com/ceph/ceph
```

Comment:
Why are we removing `cephadm_git_url`? 

I see we keep it in https://github.com/ceph/ceph/pull/57974/files#diff-b53b67177a332a28f8f9695237eefd87b958d0d1c02a905d132244944fc0f313R30

Reply:
From reef on, cephadm needs to be pulled differently. I referenced this commit to make the change: https://github.com/ceph/ceph/commit/967716f2e6c8cd3f7d810a5f8e5e49448717e513

Quincy continues to be pulled via `cephadm_git_url` and `cephadm_branch`.